webapp/channels/src/components/searchable_channel_list.tsx

```diff
@@ -371,52 +403,74 @@ export default class SearchableChannelList extends React.PureComponent<Props, St
             </div>
         );
 
-        let channelDropdown;
-        let checkIcon;
+        const checkIcon = (
+            <CheckIcon
+                size={18}
+                color={'var(--button-bg)'}
+            />
+        );
+        const channelDropdownItems = [
+            <Menu.ItemAction
+                key='channelsMoreDropdownAll'
+                id='channelsMoreDropdownAll'
+                onClick={() => this.props.changeFilter(FILTER.all)}
+                icon={<GlobeIcon size={16}/>} //  todo use the correct icon from compass: PR opened: https://github.com/mattermost/compass-icons/pull/78/files
```

Comment:
Attention

Reply:
@M-ZubairAhmed before doing that I think compass icons package needs to be published first. I dont think I can do it as community member.

Reply:
@M-ZubairAhmed I see that Matt has published a new version if compass icon package. I upgraded the package and used the correct icon